src/core/test/system/lender/base/redeem_two_tranche.sol

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2020 Centrifuge
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.12;
+
+import "../../system.t.sol";
+
+contract RedeemTwoTrancheTest is SystemTest {
+
+    WhitelistOperator jOperator;
+    WhitelistOperator sOperator;
+
+    DefaultDistributor dDistributor;
+
+    Investor juniorInvestor;
+    address  juniorInvestor_;
+
+    Investor seniorInvestor;
+    address  seniorInvestor_;
+
+    function setUp() public {
+        bytes32 operator_ = "whitelist";
+        bytes32 distributor_ = "default";
+        baseSetup(operator_, distributor_, true);
```

Comment:
Is the baseSetup setting up one or two tranches by default?

Reply:
Okay, now I understand. 
It would be easier to read
```javascript
bool deploySeniorTranche = true;
baseSetup(operator_, distributor_,  deploySeniorTranche);
```